relayer/store/store.go

```diff
@@ -0,0 +1,190 @@
+package store
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jinzhu/gorm"
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+)
+
+type Status int
+
+const (
+	CommitmentWitnessed            Status = iota // 0
+	InitialVerificationTxSent      Status = iota // 1
+	InitialVerificationTxConfirmed Status = iota // 2
+	ReadyToComplete                Status = iota // 3
+	CompleteVerificationTxSent     Status = iota // 4
+)
+
+type BeefyItem struct {
```

Comment:
BeefyItem -> BeefyWitnessRelay ? or any better name suggestions?

Reply:
BeefyRelayInfo?

Reply:
It's a database item that contains the information required throughout the relay lifecycle:
- Marshaled original justification contents (ValidatorAddresses, SignedCommitment)
- Lifecycle info i.e. Status, Tx hashes, Seed, etc.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/05ed130b01538ab8198d39a72e0e594352c4ae24